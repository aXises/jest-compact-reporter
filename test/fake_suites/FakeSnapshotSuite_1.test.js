it(`fails snapshot`, () => {
  expect({
    asd: 23,
    abc: "smthNew"
  }).toMatchSnapshot();
});