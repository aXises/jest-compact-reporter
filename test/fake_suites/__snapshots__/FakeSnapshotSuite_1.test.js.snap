// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fails snapshot 1`] = `
Object {
  "abc": "smthNew",
  "asd": 23,
}
`;
